tools/check/check.go

```diff
@@ -366,10 +366,12 @@ func populateConfig() error {
 }
 
 func main() {
-	logger.Init("", *verbose, false, os.Stderr)
+	logger.Init("", false, false, os.Stderr)
```

Comment:
Since we are logging info, the output should be to os.Stdout

Reply:
Done